client/src/pages/Search.js

```diff
@@ -121,16 +122,30 @@ class Search extends Page {
 		showSaveSearch: false,
 	}
 
-	getSearchPart(type, query, pageNum = 0, pageSize = 10) {
+	getPaginatedNum = (part, pageNum = 0) => {
+		let goToPageNum = pageNum
+		if (part !== undefined) {
+			goToPageNum = part.pageNum
+		}
+		return goToPageNum
+	}
+
+	getPaginated = type => {
 		const { pagination } = this.props
 		const typeLower = type.toLowerCase()
 		const pageLabel = this.pageLabel(typeLower)
-		const paginatedPart = pagination[pageLabel]
+		return  pagination[pageLabel]
+	}
 
+	pageLabel = (type, prefix = this.componentPrefix) => {
+		return `${prefix}${type}`
+	}
+
+	getSearchPart(type, query, pageNum = 0, pageSize = 10) {
+		const typeLower = type.toLowerCase()
 		let subQuery = Object.assign({}, query)
-		subQuery.pageNum = (paginatedPart === undefined) ? pageNum : paginatedPart.pageNum
+		subQuery.pageNum = pageNum //(paginatedPart === undefined) ? pageNum : paginatedPart.pageNum
```

Comment:
Please remove the commented code at the end of the line.

Reply:
Done